pkgs/servers/web-apps/engelsystem/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  stdenv
+, fetchzip
+, php
+}:
```

Comment:
There are two styles in widespread use in Nixpkgs

```nix
{ stdenv
, fetchzip
, php
}:
```

```nix
{ stdenv, fetchzip, php }:
```

Please use one of these styles.

Reply:
fix the style